index.html

```diff
@@ -72,9 +83,160 @@
+                has been the
+                industry's standard
+                dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <div class="recipient">
+                <ul>
+                    <li>
+                        <div>
+                            <h3>css</h3>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="verticale"></div>
+                    </li>
+                    <li>
+                        <div>
+                            <h3>html</h3>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="verticale"></div>
+                    </li>
+                    <li>
+                        <div>
+                            <h3>bootstrap</h3>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="verticale"></div>
+                    </li>
+                    <li>
+                        <h3>Ruby</h3>
+                    </li>
+                </ul>
+            </div>
+            <a href=#>
+                <div class="button">
+                    <h3>See Project</h3>
+                </div>
+            </a>
+        </div>
+        <div class="project first">
+            <div class="preview"></div>
+            <h1 class="description_title">Multi-Post Stories</h1>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                has been the
+                industry's standard
+                dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <div class="recipient">
+                <ul>
+                    <li>
+                        <div>
+                            <h3>css</h3>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="verticale"></div>
+                    </li>
+                    <li>
+                        <div>
+                            <h3>html</h3>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="verticale"></div>
+                    </li>
+                    <li>
+                        <div>
+                            <h3>bootstrap</h3>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="verticale"></div>
+                    </li>
+                    <li>
+                        <h3>Ruby</h3>
+                    </li>
+                </ul>
+            </div>
+            <a href=#>
+                <div class="button">
+                    <h3>See Project</h3>
```

Comment:
- [ ] This should be a `button` tag, that you will use to show up the project detail in future projects, I recommend you to change the `h3` tag to `button` tag with `type=" button"`

- [ ] The button should be similar to the design as you can see in screenshots below, Kindly stick to the design and work more on the button style

## Figma's 
![real](https://user-images.githubusercontent.com/80895497/153662085-ce40ed07-9225-459a-8c4b-9fd43e3f7631.PNG)

## Yours
![design](https://user-images.githubusercontent.com/80895497/153662153-fb8094dc-78d2-4c89-9b0b-c0d1cf17c65a.PNG)